### ISSUE TITLE
Own file extension -- .htmlz for example

### ISSUE THREAD

**Issue by kramola-RU** [gildas-lormeau/SingleFileZ#15]
due the fact the saved files can only be opened via SF-addon...... it might be convenient to have unique extension for these files

additional question:
may be it possible to make saved files opens in IE (without SFz)?

**Comment by gildas-lormeau** [comment]
You can do that, choose any extension you would like by editing the option File name > template.

You don't need to install SingleFileZ on Firefox to view saved pages. You can also view them in Safari by selecting the option "Disable Local File Restrictions" in the "Develop" menu. Only Chrome requires SingleFileZ to be installed. That's why I decided to use .html as filename extension by default. I did not test to saved pages in IE yet but it's not really a priority for me... The most important thing is that in 10-20 years, you'll be able to open these files and I hope you won't be using IE anymore ;)

**Comment by kramola-RU** [comment]
OK
I see

**Comment by gildas-lormeau** [comment]
For your information, I added this question in the FAQ: https://github.com/gildas-lormeau/SingleFileZ/blob/master/faq.md#why-singlefilez-uses-html-as-the-default-filename-extension.

**Comment by gildas-lormeau** [comment]
I'm closing this issue because I updated the FAQ to explain the reason why SingleFileZ uses by default the `.html` extension. Feel free to re-open it if necessary.

**Comment by gildas-lormeau** [comment]
TL;DR The user experience is better from my point view for Firefox users with the `.html` extension. Using `.htmlz` would penalize people that don't have SingleFileZ installed (on Firefox) because they would not be able to open the file by double-clicking it for example.
